documentation/ci-cd-strategy.md

```diff
@@ -0,0 +1,59 @@
+# Find, API and publish CI/CD
+
+Today the deployment process is somewhat manual, unreliable and slow. This creates frustration for the devs and it takes up a lot of their time. As we are moving to GOV.UK paas and Github Actions, we can take the opportunity to move to a continuous deployment model, but with the right controls in place. Here is the proposal.
```

Comment:
Overtime, the strategy is to move to Gov.UK PaaS, using Github Actions as an orchesteration deployment tool. This would allow us to move to a continous deployment model, anchored by quality integrated testing.

Reply:
Done, thanks